src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ExternalReference from 'src/shared/components/external-reference/ExternalReference';
+import MainAccordion from './MainAccordion';
+import PublicationsAccordion from './PublicationsAccordion';
+
+import {
+  getEntityViewerSidebarPayload,
+  getEntityViewerSidebarUIState
+} from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors';
+import { updateEntityUI } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarActions';
+
+import { RootState } from 'src/store';
+import JSONValue from 'src/shared/types/JSON';
+import { EntityViewerSidebarPayload } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarState';
+
+import styles from './GeneOverview.scss';
+
+type Props = {
+  sidebarPayload: EntityViewerSidebarPayload | null;
+  sidebarUIState: { [key: string]: JSONValue } | null;
+  updateEntityUI: (uIstate: { [key: string]: JSONValue }) => void;
+};
+
+const GeneOverview = (props: Props) => {
+  if (!props.sidebarPayload) {
+    return null;
+  }
+  const { gene } = props.sidebarPayload;
+  return (
+    <div>
+      <div className={styles.geneDetails}>
+        <span className={styles.geneSymbol}>{gene.symbol}</span>
+        <span className={styles.stableId}>{gene.id}</span>
+      </div>
+
+      <div className={styles.sectionHead}>Gene name</div>
+
+      <div className={styles.geneName}>
+        <ExternalReference
+          label={gene.name}
+          linkText={gene.source?.value || ''}
```

Comment:
Gene should not have a source. The source is a property of a metadata item, which, in this case, is gene's name (the HGNC link seen in the pdf is a naming authority for genes). I'll ask Kieron where he thinks the source should go in this case.

Reply:
Talked to Kieron. He suggests putting extended name data in the `metadata` field, like so:

```
{
  "metadata": {
    "name": {
      "value": "HGNC:1101",
      "description": "BRCA2 DNA repair associated",     
      "source_uri": "https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:1101",
      "source": {
        "name": "Human Gene Naming Consortium",
        "url": "https://www.genenames.org"
      }
    }
  }
}
```
the shape above is very provisional; the actual names of the fields can change (`value` and `description` are particularly suspicious), but at least it moves the `source` from the top level of the gene, and co-locates the source with the data that it is the source for.

Reply:
I have now moved `source` to be inside `metadata.name`